src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -1151,6 +1151,7 @@ enum OpcodeServer : uint32
     SMSG_ITEM_PURCHASE_REFUND_RESULT                  = 0x2535,
     SMSG_ITEM_PUSH_RESULT                             = 0x25BA,
     SMSG_ITEM_TIME_UPDATE                             = 0x2714,
+    SMSG_ITEM_UPGRADE_RESULT                     = 0x25DB,
```

Comment:
0x25DB does not have any handler in 6.2.4 client - sending it will not affect anything
Since it does not have a handler, please place this line at the bottom of the enum, just after SMSG_ACCOUNT_HEIRLOOM_UPDATE. Reason for this is that the entire enum of opcodes that have client handlers is generated automatically on every patch so anything unhandled there will simply get lost.